duradmin/src/main/webapp/WEB-INF/jsp/spaces/spaces-manager.jsp

```diff
@@ -279,7 +279,7 @@
             <h1>Space Detail</h1>
             <h3 class="object-name">Space Name Here</h3>
             <div class="button-bar">
-              <!-- 
+              <!--
```

Comment:
This formatting change probably should be reverted or at least added as a separate formatting themed commit since it is not related to duracloud-1214 (as far as I can tell).